packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/template/FluentClientMethodTemplate.java

```diff
@@ -213,6 +215,30 @@ protected void generatePagedAsyncSinglePage(ClientMethod clientMethod, JavaType
         }
     }
 
+    @Override
+    protected void pagedSinglePageResponseConversion(ProxyMethod restAPIMethod, ClientMethod clientMethod,
```

Comment:
generated code: 
```java
LiveEventGetStatusResult result = this.client
    .<LiveEventGetStatusResult, LiveEventGetStatusResult>getLroResult(res, LiveEventGetStatusResult.class,
        LiveEventGetStatusResult.class, Context.NONE)
    .getFinalResult();
return new PagedResponseBase<>(res.getRequest(), res.getStatusCode(), res.getHeaders(), result.value(), null,
    null);
```

Reply:
It's not easy. `nextLink` would also need the result.

Renamed to `lroPageableResult`.